micrometer/src/test/java/org/jboss/eap/qe/micrometer/multiple/BaseMultipleTestCase.java

```diff
@@ -0,0 +1,48 @@
+package org.jboss.eap.qe.micrometer.multiple;
+
+import jakarta.ws.rs.client.Client;
+import jakarta.ws.rs.client.ClientBuilder;
+import jakarta.ws.rs.client.WebTarget;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.jboss.eap.qe.micrometer.container.PrometheusMetric;
+import org.jboss.eap.qe.ts.common.docker.Docker;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+
+public abstract class BaseMultipleTestCase {
```

Comment:
My understanding is that this class is not copy pasted from WF, right? So maybe can you please add some short javadoc here?

Reply:
All of them are all copied from WildFly, but I might add some headers and Javadoc as well. Fixing.

Reply:
Thank you!